composer.json

```diff
@@ -87,11 +91,11 @@
         "hoa/file": "1.16.01.15",
         "sentry/sentry-symfony": "^2.0",
         "danielstjules/stringy": "~3.1",
-        "burgov/key-value-form-bundle": "@stable",
+        "burgov/key-value-form-bundle": "^1.4",
         "nelmio/api-doc-bundle": "^2.13",
         "nelmio/cors-bundle": "^1.4",
-        "facebook/facebook-instant-articles-sdk-php": "v1.5.*",
-        "takeit/amp-html-bundle": "^0.2.0",
+        "facebook/facebook-instant-articles-sdk-php": "dev-more_php_versions",
+        "takeit/amp-html-bundle": "dev-master",
```

Comment:
this is risky, better to release a new version

Reply:
There is already a newly released version, just has to be used. It wasn't released at the time this PR was created.